pkgs/data/icons/sweet-folders/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
```

Comment:
I'm having a hard time building the package, because it's not in `pkgs/by-name` and not included in `pkgs/top-level/all-packages.nix`. I think it would be best if you moved it to `pkgs/by-name/sw/sweet-folders/package.nix`

Reply:
So also themes and icons themes should go under `by-name`?

Reply:
Everything is moving towards `by-name` these days. There's the [RFC 140](https://github.com/NixOS/rfcs/pull/140) and [`pkgs/by-name/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md) which gives some more details, but the gist of it is this:

- New packages should go to `by-name/<first two letters of package name>/<package>/package.nix`
- It will then get automatically included, without you having to edit `pkgs/top-level/all-packages.nix` (with some exceptions)

Reply:
@h7x4 fixed.